.github/workflows/test-build-windows.yml

```diff
@@ -209,3 +209,157 @@ jobs:
           cd ..\anomaly-detection
           dir
           .\gradlew.bat :integTestRunner --tests "com.amazon.opendistroforelasticsearch.ad.rest.*IT" --tests "com.amazon.opendistroforelasticsearch.ad.e2e.*IT" -D tests.rest.cluster=localhost:9200 -D tests.cluster=localhost:9200 -D tests.clustername=es-integrationtest --stacktrace
+
+  Test-AD-plugin-With-Security:
+    needs: [build-es-artifacts, build-kibana-artifacts]
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Retrieve plugin tags
+        run:  echo ::set-env name=p_tag_ad::$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/anomaly-detection)
+        shell: bash
+      - uses: actions/checkout@v1
+        with:
+           repository: opendistro-for-elasticsearch/anomaly-detection
+           ref: ${{env.p_tag_ad}}  
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Set Up JDK 
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+        
+      - name: RUN ES and AD IntegTest
+        run: |
+          dir
+          python -m pip install --upgrade pip
+          echo pip3 -version
+          pip3 install awscli
+          $PACKAGE="opendistroforelasticsearch"
+          $OD_VERSION=$(python ./bin/version-info --od)
+          $S3_PACKAGE="odfe-"+$OD_VERSION+".zip"
+          dir
+          echo downloading zip from S3 
+          aws s3 cp s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/odfe-windows/staging/odfe-window-zip/$S3_PACKAGE .\
+          echo "unzipping $S3_PACKAGE"
+          unzip .\$S3_PACKAGE
+          echo "running es"
+          nohup .\$PACKAGE-$OD_VERSION\bin\elasticsearch.bat &
+          echo "Waiting for 2min"
```

Comment:
Are we doing anything here to wait for 2min? I mean like in "sleep 2m"?

Reply:
just copy from previous IT case. It's using "ping -n 120 127.0.0.1 >.\out.txt" for 2m sleeping.

Reply:
Good catch. That was there in the previous tests as well.
It is probably the reason that led to failure cases in workflows.
Agree we can add some 120 seconds sleep on each test cases.